tests/World/test_Ambient.py

```diff
@@ -1,5 +1,26 @@
 from adr.World import Ambient
+import numpy.testing as npt
+import pytest
 
 
-def test_instantiation():
-    pass
+@pytest.fixture
+def base_ambient():
+    base_ambient = Ambient(temperature=288.15, pressure=101325, humidity=30)
+    return base_ambient
+
+
+@pytest.fixture
+def base_ambient_ERROR():
+    base_ambient_ERROR = Ambient(temperature=288.15, pressure=101325, humidity=-30)
+    return base_ambient_ERROR
```

Comment:
You can test for errors on a test_instantiation_with_error(), not using the fixtures, like so:

```
ambient = Ambient(temperature=0)
with raises(ValueError) as e_info:
    air_density = ambient.air_density

ambient = Ambient(humidity=104)
with raises(ValueError) as e_info:
    air_density = ambient.air_density
```

When you are inside a "with raises(Error)" context pytest will verify if the code inside it (air_density = ambient.air_density) raises the given error. If it does, the test passes, if not, it won't.